sycl/source/detail/scheduler/commands.hpp

```diff
@@ -121,7 +120,7 @@ class Command {
   };
 
   Command(
-      CommandType Type, QueueImplPtr Queue,
+      CommandType Type, queue_impl *Queue,
```

Comment:
@sergey-semenov or @intel/sycl-graphs-reviewers , do you know if `Queue` must be non-null for any of the subclasses?

Reply:
I believe it must be not-null for all of them. even host task.

Reply:
Even if we're submitting to the graph and not actual queue?

Reply:
graph has queue as input parameter for begin_recording API https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc#84-graph,
so I expect graph commands to have corresponding queue too. 
Although it is better to have a confirmation from graph extension authors or check yourself.

Reply:
Yeah, since sycl buffers are separated into initialization commands and copy commands in the scheduler, we make sure that those have completed when it comes to finalize graph. We've got a bit of info on that in the last 3 paragraphs of https://github.com/intel/llvm/blob/sycl/sycl/doc/design/CommandGraph.md#scheduler-integration